entity-service/src/main/kotlin/com/egm/stellio/entity/authorization/Group.kt

```diff
@@ -0,0 +1,37 @@
+package com.egm.stellio.entity.authorization
+
+import com.egm.stellio.shared.util.AuthContextModel.AUTH_REL_IS_MEMBER_OF
+import com.egm.stellio.shared.util.JsonLdUtils
+import com.egm.stellio.shared.util.JsonLdUtils.NGSILD_NAME_PROPERTY
+import java.net.URI
+
+data class Group(
+    val id: URI,
+    val type: List<String>,
```

Comment:
`type` is known, you can hardcode it (as is done for subscriptions for instance)